packages/expo-build-properties/src/ios.ts

```diff
@@ -6,6 +6,10 @@ const { createBuildPodfilePropsConfigPlugin } = IOSConfig.BuildProperties;
 
 export const withIosBuildProperties = createBuildPodfilePropsConfigPlugin<PluginConfigType>(
   [
+    {
+      propName: 'expo.newArchEnabled',
+      propValueGetter: (config) => config.ios?.newArchEnabled?.toString(),
```

Comment:
From my understanding we need `?.toString()` because Config Plugins `propValueGetter` requires a string to be returned 

https://github.com/expo/expo/blob/0299d154da99bce511f9a7db6821d74ef63a1805/packages/@expo/config-plugins/build/utils/BuildProperties.types.d.ts#L20-L25

Reply:
oh you are right. so the config-plugin only supports strings. the *Podfile.properties.json*, even it's json format, the final outcome would be `{ "newArchEnabled": "true" }`.
we'd better update the Podfile change like following comment